os/linux/local/mode/pendingupdates.pm

```diff
@@ -102,23 +112,20 @@ sub check_options {
     }    
 }
 
```

Comment:
It would be cool to directly return an UNKNOWN with a clear message explaining that --check-security is only available with rhel.  (if defined check_security && os mode not rhel ...)